shenyu-admin/src/main/java/org/apache/shenyu/admin/listener/consul/ConsulDataInit.java

```diff
@@ -25,7 +25,10 @@
 
 /**
  * The type Consul data init.
+ *
+ * @deprecated replace with {@link ConsulDataChangedInit}
  */
+@Deprecated
```

Comment:
why add Deprecated

Reply:
Replaced with `ConsulDataChangedInit`, this class will no longer be used.

Reply:
this used by  consul sycn data？

Reply:
yes.